tests/kafkatest/tests/streams/streams_broker_bounce_test.py

```diff
@@ -243,8 +230,9 @@ def test_broker_type_bounce(self, failure_mode, broker_type, sleep_time_secs, nu
     @cluster(num_nodes=7)
     @matrix(failure_mode=["clean_shutdown"],
             broker_type=["controller"],
-            sleep_time_secs=[0])
-    def test_broker_type_bounce_at_start(self, failure_mode, broker_type, sleep_time_secs):
+            sleep_time_secs=[0],
+            metadata_quorum=[quorum.isolated_kraft])
```

Comment:
I thought we agree to run all KS system test with `combined_kraft` ?

Reply:
Maybe I got my wires crossed, I thought we wanted to go back to how they were originally, same applies to all others below.